queuelib/queue.py

```diff
@@ -143,7 +144,7 @@ def _cleanup(self):
         for x in glob.glob(os.path.join(self.path, 'q*')):
             os.remove(x)
         os.remove(os.path.join(self.path, 'info.json'))
-        if not os.listdir(self.path):
+        with suppress(OSError):
```

Comment:
This is equivalent according to [the docs for `os.rmdir`](https://docs.python.org/3/library/os.html#os.rmdir):

> If the directory does not exist or is not empty, an FileNotFoundError or an OSError is raised respectively.